### PR TITLE
185 header

### PR DESCRIPTION
**Pull request by mikes-lunarg**
A few quirks with this one, I tried to split the noisier rename change into its own commit.

@jeremyg-lunarg Please take a look at the `sync_val_gen.py` changes. I think we will probably need another gitlab issue like https://gitlab.khronos.org/vulkan/vulkan/-/issues/2745 for VK_HUAWEI_invocation_mask, but wanted to double check with you first.

### CODE REVIEW

File: scripts/sync_val_gen.py [hunk 1]
```diff
@@ -65,8 +65,8 @@ def ParseAccessMasks(valid_usage_path, all_stages):
         'VK_PIPELINE_STAGE_2_ALL_COMMANDS_BIT_KHR',
         # expands to INDEX_INPUT and VERTEX_ATTRIBUTE_INPUT, which are included explicitly in the VUIDs
         'VK_PIPELINE_STAGE_2_VERTEX_INPUT_BIT_KHR',
-        # temporarily disable VK_HUAWEI_subpass_shading until it has stabilized
-        'VK_PIPELINE_STAGE_2_SUBPASS_SHADING_BIT_HUAWEI',
+        # temporarily disable VK_HUAWEI_invocation_mask pending more detail in the sync chapter
+        'VK_PIPELINE_STAGE_2_INVOCATION_MASK_BIT_HUAWEI',
```
Comment (jeremyg-lunarg) [inline scripts/sync_val_gen.py:69]: wow, another one?

**Review by jeremyg-lunarg — COMMENTED** [review]
The sync_val_gen.py changes the new code it generate look good to me. Its a shame we got another pipeline stage without the relevant details.

**Comment by jeremyg-lunarg** [comment]
As far as if we need to open a working group issue, it is hard to tell how far along they are with this extension. There's no valid usage for the pipeline stage either. So we can either open a copy & paste of the last issue or wait a few more header versions to see if there's more in the pipeline.

File: scripts/parameter_validation_generator.py [hunk 2]
```diff
@@ -528,11 +528,10 @@ def endFile(self):
                             table_type = 'device'
                         else:
                             print("Error in parameter_validation_generator.py CodeGen.")
-                        norm_ext_name = ext_name_define[:-15].lower()
```
Comment (ncesario-lunarg) [inline scripts/parameter_validation_generator.py:531]: Nice :)

**Review by ncesario-lunarg — APPROVED** [review]

**Pull request merged**